handbook/engineering/campaigns/index.md

```diff
@@ -47,11 +47,22 @@ When I create a campaign to make large-scale code changes I want to _focus on th
 
 For each iteration (currently one month long), we follow this process:
 
-* Before the iteration begins, each team member prepares for the iteration planning meeting by considering what they would like to see in the next iteration, and by looking over [our backlog](https://github.com/sourcegraph/sourcegraph/labels/team%2Fcampaigns) and setting the Milestone for issues we'd like to include.
-* We then have our planning meeting to determine our common goals for the iteration. We create a [tracking issue](https://about.sourcegraph.com/handbook/engineering/tracking_issues) for the iteration which will hold our goals and plans, in addition to tracking the issues we intend to address.
-* Each Monday, the team meets with stakeholders to stay in sync, and to set our personal goals for the week. We record these goals, and the previous week's progress, as status update comments on the tracking issue. The engineering manager then compiles these into a summary which is sent to the team and stakeholders.
+* Before the iteration begins, we do pre-planning to make the most of our the planning meeting:
+  * EM creates a [tracking issue](https://about.sourcegraph.com/handbook/engineering/tracking_issues) for the iteration which will hold our goals and plans, in addition to tracking the issues we intend to address.
```

Comment:
And perhaps the milestone for N+1 and a label `planned/CampaignsSprintN`?

Reply:
Good ideas!